lib/printer_info.dart

```diff
@@ -3100,7 +3100,7 @@ class Printer {
       "printerId": mPrinterId,
       "printInfo": mPrinterInfo.toMap(),
       "key": key,
-      "encode":key
```

Comment:
Great catch @rendesg! Thanks for the PR.

Reply:
You're welcome, thank you for the library ;)